spring-integration-core/src/main/java/org/springframework/integration/StaticMessageHeaderAccessor.java

```diff
@@ -120,4 +123,20 @@ public static <T> T getSourceData(Message<?> message) {
 		return (T) message.getHeaders().get(IntegrationMessageHeaderAccessor.SOURCE_DATA);
 	}
 
+	/**
+	 /**
+	 * Get a {@link ContextView} header if present.
+	 * @param message the message to get a header from.
+	 * @return the {@link ContextView} header if present.
+	 * @since 6.0.5
+	 */
```

Comment:
```suggestion
	/**
	 * Get a {@link ContextView} header if present.
	 * @param message the message to get a header from.
	 * @return the {@link ContextView} header if present.
	 * @since 6.0.5
	 */
```